DB/GADPerson.m

```diff
@@ -95,6 +101,50 @@ -(GADPerson *)initWithDictionary: (NSDictionary *) dict {
     return people;
 }
 
++ (void) fetchPersonInfoWithCriteria:(NSDictionary*)crit andUsername:(NSString*)usrname andPassword:(NSString*) pw completionHandler:(void(^_Nonnull)(NSArray<GADPerson *> *))completion {
+
+    NSMutableArray * queryItems = [NSMutableArray array];
```

Comment:
Let's use generics! Try `NSMutableArray<NSURLQueryItem *> *queryItems = [NSMutableArray<NSURLQueryItem *> new];`

This way, we have compile-time typechecking for the array. It prevents bugs!